### PR TITLE
#50 test-for-app-range

### LINKED ISSUE #50: (SP: 1) Write tests for range component
Writing tests don't forget to mock elements if neccessary.

Test cases:

- [x] it should renders correctly
- [x] it should call onChange when slider is moved
- [x] it should call onChange when input is changed
- [x] it should not call onChange when input is changed wit not a number
- [x] it should call onChange whith min number if input is empty
- [x] it should update prices when input is blurred and input is greater than max value
- [x] it should not update prices when input is blurred and value in input has not changed

### PR DESCRIPTION
**Pull request by pavlo1377**
Tests for AppRange component

image

## Summary by CodeRabbit

- **Tests**
  - Added comprehensive unit tests for the range selection component, covering rendering, user interactions, input validation, and boundary conditions.

### CODE REVIEW

**Review by oksana0145 — APPROVED** [review]

**Review by yrudikov — APPROVED** [review]

**Review by NataliiaDanchenko — APPROVED** [review]

**Review by VoroninaMaria — APPROVED** [review]

File: src/tests/unit/components/app-range/AppRange.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
+vi.mock('~/hooks/use-debounce', () => ({
+  useDebounce: (fn) => fn
+}))
+
+import { fireEvent, render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { beforeEach, describe, expect, it, vi } from 'vitest'
+import AppRange from '~/components/app-range/AppRange'
```
Comment (vlad-khrychov) [inline src/tests/unit/components/app-range/AppRange.spec.jsx]:
```suggestion
import { fireEvent, render, screen } from '@testing-library/react'
import userEvent from '@testing-library/user-event'
import { beforeEach, describe, expect, it, vi } from 'vitest'
import AppRange from '~/components/app-range/AppRange'

vi.mock('~/hooks/use-debounce', () => ({
  useDebounce: (fn) => fn
}))
```

**Review by vlad-khrychov — APPROVED** [review]
